src/pages/map.tsx

```diff
@@ -109,7 +75,9 @@ interface Props {
 const MapPage = ({ schedule }: Props) => {
   const [map, setMap] = React.useState<MapboxGl.Map | undefined>(undefined);
   const [searchBy, setSearchBy] = React.useState<SearchFilter>('kecamatan');
+  const [activeLoc, setActiveLoc] = React.useState<LocationData | undefined>(undefined);
   const [searchKeyword, setSearchKeyword] = React.useState('');
+  const { colorMode } = useColorMode();
```

Comment:
```suggestion
  const mapFlyoutBackgroundColor = useColorModeValue('white', 'gray.900');
```

You can use `useColorModeValue()` instead so you can use the colours immediately without any other checks.

Reply:
updated thanks for this